Utils/hdinsight-node-common/pom.xml

```diff
@@ -88,6 +88,18 @@
                     <target>1.8</target>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
it reported from `[ERROR]   The project com.microsoft.azuretools.wap4ej.build:parent:3.0.4-SNAPSHOT (/home/travis/build/Microsoft/azure-tools-for-java/PluginsAndFeatures/azure-toolkit-for-eclipse/pom.xml) has 1 error`